wear/build.gradle

```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.application'
+
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "22.0.1"
+
+    defaultConfig {
+        applicationId "com.eveningoutpost.dexdrip"
+        minSdkVersion 20
+        targetSdkVersion 23
+        versionCode 1
+        versionName "1.0.2"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile 'com.ustwo.android:clockwise-wearable:1.0.2'
+    compile 'com.google.android.support:wearable:1.1.0'
+    compile 'com.google.android.gms:play-services-wearable:7.3.0'
```

Comment:
I was silently worrying about the method limit.  It's a little troubling how easy it is to hit with a few libraries and Google Play Services.  I don't want to think about how much fun it would be to get ProGuard working, but it might be necessary sooner rather than later.  We could probably get away for a while by copying/porting some code out of the android-uploader-core library.

I think the newer version of play services had problems with "legitimate" Android Wear watches.  I was hoping someone had ideas on that.